app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -23,9 +23,9 @@
     android:id="@+id/story_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="20dp"
+    android:layout_marginStart="28dp"
     android:layout_marginTop="12dp"
-    android:layout_marginEnd="20dp"
+    android:layout_marginEnd="28dp"
     android:layout_marginBottom="12dp"
     app:cardBackgroundColor="@color/white"
```

Comment:
In the child linear layout where there is margin on all sides (Lines 42-45) the margin start/end should be different because the text `In Progress` is at `12dp` margin from card. I think this might lead to changes in margin/padding in various widgets.

Reply:
Ditto in landscape layout

Reply:
Margin start is 14dp for the segmented progress and for textview it is 12dp. So to make consistent I kept 12 dp. Otherwise I have to set 16dp for segmented progress.

Reply:
> Margin start is 14dp for the segmented progress and for textview it is 12dp. So to make consistent I kept 12 dp. Otherwise I have to set 16dp for segmented progress.
Also If you check entire layout of segemented progress and textview its gives 12dp  margin from start.

Reply:
Yeah, this makes sense, lets keep it 12dp then.

Reply:
Okay.